Booking.Repository/Implementations/BookingConfirmationsRepository.cs

```diff
@@ -26,5 +26,10 @@ public async Task<List<BookingConfirmation>> GetAllAsync()
 
             return bookingConfirmations;
         }
+
+        public Task GetAsync(string entity)
+        {
+            throw new NotImplementedException();
```

Comment:
Why does it not implemented?

Reply:
I've corrected.
![image](https://user-images.githubusercontent.com/80592971/180664204-0111f7ac-9936-4067-8e60-818c4e5644f1.png)